packages/ui/src/utils/numberHelpers.ts

```diff
@@ -0,0 +1,173 @@
+import JSBI from "jsbi";
```

Comment:
For the new utils, we already have a `formatDecimalValues` and a `formatBalance`. You should move those functions here and consolidate them with the functions you added if possible. I think you could move all display related helpers to this ui package.